pkg/scclient/client.go

```diff
@@ -40,6 +42,8 @@ const (
 	DefaultRetryTimeout = 500 * time.Millisecond
 	HeaderRevision      = "X-Resource-Revision"
 	EnvProjectID        = "CSE_PROJECT_ID"
+	//EnvCheckSCIInterval sc instance health check interval in second
+	EnvCheckSCIInterval = "CHASSIS_SC_HEALTH_CHECK_INTERVAL"
```

Comment:
https://docs.go-chassis.com/user-guides/env.html add env in this doc

Reply:
done.